websites/L/Lazada/presence.ts

```diff
@@ -0,0 +1,71 @@
+const presence = new Presence({
+		clientId: "1201520071055515698",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.ibb.co/LpxnMfj/512x512pngoflazada.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname } = document.location;
+
+	let item: HTMLElement;
```

Comment:
Why is the item declared in a let, when it's set in an if case and then only used in the same if case...

Reply:
I'm using `let` for future changes. Also, I have tried using other variable declaration and it's giving erros.

Reply:
You can define variables within scopes such as if statements.
```ts
if (...) {
  const item = ...;
} else if (...) {
  const item = ...;
}
```

Reply:
May I use `document.querySelector(".JrAyI").textContent;` instead?

Reply:
Yes. For that selector specifically, if there is an alternative non computer-generated selector, that would be preferred. If not possible, that's fine too

Reply:
I was trying to find a way yesterday, but when I do, I’ll just send another pull request for improvements. Thank you.